CODEOWNERS

```diff
@@ -63,3 +63,6 @@ src/App.js  @benisgold @skylarbarrera @jinchung
 metro.config.js @skylarbarrera @benisgold 
 metro.transform.js @skylarbarrera @benisgold 
 react-native.config.js  @skylarbarrera @benisgold
+
+# SWAPS V2
+src/__swaps__ @walmat @dereknelson
```

Comment:
this is gonna make reviews pretty painful you may wanna leave this out until dev is complete

Reply:
fair, will revert this commit